GitVersionCore.Tests/Mocks/MockCommitLog.cs

```diff
@@ -8,7 +9,10 @@ public class MockCommitLog : ICommitLog, ICollection<Commit>
 
     public IEnumerator<Commit> GetEnumerator()
     {
-        return Commits.GetEnumerator();
+        if (SortedBy == CommitSortStrategies.Reverse)
```

Comment:
By default libgit2 starts at latest commit and goes back. Or at least seems to, I naively put reverse = oldest first. @nulltoken what should I do here